src/docs/development/packages-and-plugins/using-packages.md

```diff
@@ -34,70 +37,69 @@ and supports searching among all published packages.
+     Hot reload and hot restart only update the Dart code,
+     so a full restart of the app may be required to avoid
      errors like `MissingPluginException` when using the package.
 
-The
-['Installing']({{site.pub-pkg}}/css_colors#-installing-tab-)
+The [Installing]({{site.pub-pkg}}/css_colors#-installing-tab-)
 tab available on any package page on Pub is a handy reference for these
 steps.
 
-For a complete example, see [CSS Colors example](#css-example) below.
+For a complete example, see the [CSS Colors example](#css-example) below.
 
 ### Conflict resolution
 
-Suppose you want to use `some_package` and `other_package` in your app `counter`
-(or your own package), and both of these depend on `url_launcher`, but
-in different versions. Then we have a potential conflict. The best way to avoid this
-is for package authors to use [version
+Suppose you want to use `some_package` and `another_package` in your app,
+and both of these depend on `url_launcher`, but in different versions.
+Then you have a potential conflict.
+The best way to avoid this is for package authors to use [version
 ranges]({{site.dart-site}}/tools/pub/dependencies#version-constraints)
 rather than specific versions when specifying dependencies.
 
 ```yaml
 dependencies:
-  url_launcher: ^0.4.2    # Good, any 0.4.x with x >= 2 will do.
-  image_picker: '0.1.1'   # Not so good, only 0.1.1 will do.
+  url_launcher: ^0.4.2    # Good, any 0.4.x version where x >= 2 works.
+  image_picker: '0.1.1'   # Not so good, only version 0.1.1 works.
 ```
 
-If `some_package` declares the dependencies above and `other_package`
-declares a compatible  `url_launcher` dependency like `'0.4.5'` or `^0.4.0`,
-`pub` is able to resolve the issue automatically. Similar
-remarks apply to plugin packages' platform-specific dependencies on
-[Gradle modules][] and/or [CocoaPods][].
+If `some_package` declares the dependencies above and `another_package`
+declares a compatible `url_launcher` dependency like `'0.4.5'` or
+`^0.4.0`, Pub resolves the issue automatically.
+Platform-specific dependencies on [Gradle modules][] and/or [CocoaPods][]
+are solved in a similar way.
 
-Even if `some_package` and `other_package` declare incompatible versions for
-`url_launcher`, it may still be that they actually use `url_launcher` in
-compatible ways. Then the conflict can be dealt with by adding
-a dependency override declaration to the `pubspec.yaml` file in `counter`,
+Even if `some_package` and `another_package` declare incompatible versions
+for `url_launcher`, they might actually use `url_launcher` in
+compatible ways. In this situation, the conflict can be resolved by adding
+a dependency override declaration to the app's `pubspec.yaml` file,
 forcing the use of a particular version.
 
-Forcing the use of `url_launcher` version `0.4.3` in `counter/pubspec.yaml`:
+To force the use of `url_launcher` version `0.4.3` in the app's `pubspec.yaml`:
```

Comment:
This should be a complete sentence (https://developers.google.com/style/code-samples?hl=ru#intros). Sentence fragments introducing code samples seem to occur a lot in this page.